Resources/config/serializer/Product.ProductView.xml

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<serializer>
+    <class name="Sulu\Bundle\SyliusConsumerBundle\Model\Product\ProductView" exclusion-policy="ALL">
+        <virtual-property method="getCode" serialized-name="uuid" expose="true"/>
```

Comment:
code is uuid?

Reply:
should be id and code now

thanks for the hint